src/main/java/gregtech/core/GTCETransformer.java

```diff
@@ -76,6 +78,16 @@
                     BlockVisitor.handleClassNode(classNode).accept(classWriter);
                     return classWriter.toByteArray();
                 }
+                break;
+            }
+            case RenderGlobalVisitor.TARGET_CLASS_NAME: {
+                if (FMLCommonHandler.instance().getSide() == Side.CLIENT) {
+                    ClassReader classReader = new ClassReader(basicClass);
+                    ClassWriter classWriter = new ClassWriter(ClassWriter.COMPUTE_FRAMES | ClassWriter.COMPUTE_MAXS);
+                    classReader.accept(new TargetClassVisitor(classWriter, RenderGlobalVisitor.TARGET_METHOD, RenderGlobalVisitor::new), 0);
+                    return classWriter.toByteArray();
+                }
+                break;
```

Comment:
There are serious problems here. after looking at the classbyte code, it will break the `lookupswitch` here. causing all `case` after 1010 will jump to an error op line.

i suggest to hook the `getItemById` and **return something as usual** instead of **return it yourself**